Fix compilation with socks5 USE flag.

Patch by Phobos Kappa in https://bugs.gentoo.org/762253

--- a/ext/socket/sockssocket.c	2020-12-25 05:33:01.000000000 +0200
+++ b/ext/socket/sockssocket.c	2020-12-28 15:42:50.310029778 +0200
@@ -34,7 +34,7 @@
 	init = 1;
     }
 
-    return rsock_init_inetsock(sock, host, port, Qnil, Qnil, INET_SOCKS, Qnil);
+    return rsock_init_inetsock(sock, host, port, Qnil, Qnil, INET_SOCKS, Qnil, Qnil);
 }
 
 #ifdef SOCKS5
